### PR TITLE
wip add dof range to compute range for a field

### PR DESCRIPTION
**Pull request by KristofferC**

### CODE REVIEW

File: src/Dofs/DofHandler.jl [hunk 1]
```diff
@@ -59,6 +59,13 @@ function field_offset(dh::DofHandler, field_name::Symbol)
     return offset
 end
 
+function dof_range(dh::DofHandler, field_name::Symbol)
+    f = find_field(dh, field_name)
+    offset = field_offset(dh, field_name)
+    n_field_dofs = getnbasefunctions(dh.field_interpolations[f]) * dh.fieldn_dims[f]
```
Comment (fredrikekre) [inline src/Dofs/DofHandler.jl]: `fieldn_dims` -> `field_dims`

File: src/Dofs/DofHandler.jl [hunk 2]
```diff
@@ -59,6 +59,13 @@ function field_offset(dh::DofHandler, field_name::Symbol)
...
+    return offset:n_field_dofs
```
Comment (fredrikekre) [inline src/Dofs/DofHandler.jl]: `(offset+1):n_field_dofs`

**Comment by fredrikekre** [comment]
Is this better called `field_range` (like the branch name) rather than `dof_range`?

**Comment by KristofferC** [comment]
You get the dof range for a given field. Seems good to me?

**Pull request merged**